manifests/config.pp

```diff
@@ -158,11 +158,25 @@
           content => template("rsnapshot/${script}.sh.erb"),
           mode    => '0755',
         }
-        
+
       }
     }
 
-    $cronfile = "${cron_dir}/${host}"
+    # cron on Debian seems to ignore files that have dots in their name; replace
+    # them with underscores (issue #2)
+    if $::osfamily == 'Debian' {
+      $cron_name = regsubst("${host}", '\.', '_')
```

Comment:
Yeah I didn't want to get in the middle of the work @monotek is doing!

Thanks for the module, it's great and exactly what I wanted.